concepts/components-events-groups.md

```diff
@@ -15,19 +15,19 @@ parent: Concepts
 {:toc}
 </details>
 
-There are three main structures that make up the base of a behavior-pack entity file. This doc will explain what each of them means, and how to use them.
+Three main structures make up the base of a behavior-pack entity file. This doc will explain what each of them means, and how to use them.
 
 # Components
 
-Components should be thought of as the logical building-blocks that make up minecraft-entities. All components are written by microsoft, and provided to us for use. Components can do all sorts of things, like set the size of an entity, or give it the ability to swim. The [full list of components can be found here](https://bedrock.dev/docs/stable/Entities).
+Components should be thought of as the logical building blocks that make up Minecraft-entities. All components are written by Mojang, and provided to us for use. Components can do all sorts of things, like set the size of an entity, or give it the ability to swim. The [full list of components can be found here](https://bedrock.dev/docs/stable/Entities).
 
-It is impossible to create your own components. The entire list of components is hard-coded, and provided by Microsoft. The `only` thing that should go intside the `components` object is components.
+It is impossible to create your own components. The entire list of components is hard-coded and provided by Microsoft. The `only` thing that should go inside the `components` object is components.
 
 All components begin with `minecraft:`.
 
 # Component groups
 
-Component groups should be thought of as `folders` for components. They group components together, and can be added or removed using `events` to create custom game-play.
+Component groups should be thought of as `folders` for components. The group components together and can be added or removed using `events` to create custom game-play.
```

Comment:
You introduced a typo here. `They` was correct in this context, and so was the comma.

Reply:
Alright, I'll adjust it with your suguestions.

Reply:
Let me know if the changes to my commit were good enough? or if I missed something.